src/runtime-rs/crates/hypervisor/src/device/device_manager.rs

```diff
@@ -202,13 +224,22 @@ impl DeviceManager {
         None
     }
 
-    fn get_dev_virt_path(&mut self, dev_type: &str) -> Result<Option<(u64, String)>> {
+    fn get_dev_virt_path(
+        &mut self,
+        dev_type: &str,
+        is_pmem: bool,
+    ) -> Result<Option<(u64, String)>> {
         let virt_path = if dev_type == DEVICE_TYPE_BLOCK {
-            // generate virt path
-            let current_index = self.shared_info.declare_device_index()?;
-            let drive_name = get_virt_drive_name(current_index as i32)?;
-            let virt_path_name = format!("/dev/{}", drive_name);
-
+            let (current_index, virt_path_name) = if is_pmem {
+                let current_index = self.shared_info.declare_pmem_index()?;
+                let virt_path_name = format!("/pmem/{}", current_index);
```

Comment:
It should be `format!("/dev/pmem{}", current_index);`

Reply:
fixed